app/models/job_invocation_composer.rb

```diff
@@ -194,10 +194,12 @@ def concurrency_control_params
     end
 
     def targeting_params
+      base = { :user_id => job_invocation.targeting.user_id }
```

Comment:
Maybe it would make sense to use `User.current` here instead of copying over the original user

Reply:
Definitely should be the current user